frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```
@@ -272,6 +272,8 @@
   "Container image": "Container image",
   "Cluster in STS Mode": "Cluster in STS Mode",
   "This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.": "This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.",
+  "Cluster in Azure Workload Identity / Federated Identity Mode": "Cluster in Azure Workload Identity / Federated Identity Mode",
+  "This cluster is using Azure Workload Identity / Federated Identity to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide the Client ID, Tenant ID, and Subscription ID during installation. Please see the operator description for more details.": "This cluster is using Azure Workload Identity / Federated Identity to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide the Client ID, Tenant ID, and Subscription ID during installation. Please see the operator description for more details.",
```

```suggestion
  "This cluster is using Azure Workload Identity / Federated Identity to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide the Client ID, Tenant ID, and Subscription ID during installation. Please see the operator description for more details.": "This cluster is using Azure Workload Identity / Federated Identity to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, provide the Client ID, Tenant ID, and Subscription ID during installation. See the operator description for more details.",
```
I am not sure if "please" is part of the PF guidelines, but in the RH documentation we avoid using "Please".